cmd/bad-key-revoker/main_test.go

```diff
@@ -107,16 +111,22 @@ const (
 	Unrevoked = core.OCSPStatusGood
 )
 
-func insertGoodCert(t *testing.T, dbMap *db.WrappedMap, keyHash []byte, serial string, regID int64) {
-	insertCert(t, dbMap, keyHash, serial, regID, Unexpired, Unrevoked)
+func insertGoodCert(t *testing.T, fc clock.Clock, dbMap *db.WrappedMap, keyHash []byte, serial string, regID int64) {
+	insertCert(t, fc, dbMap, keyHash, serial, regID, Unexpired, Unrevoked)
 }
 
-func insertCert(t *testing.T, dbMap *db.WrappedMap, keyHash []byte, serial string, regID int64, expiredStatus ExpiredStatus, status core.OCSPStatus) {
+func insertCert(t *testing.T, fc clock.Clock, dbMap *db.WrappedMap, keyHash []byte, serial string, regID int64, expiredStatus ExpiredStatus, status core.OCSPStatus) {
```

Comment:
nit: feels weird for these two methods to have argument order `t, fc, dbMap` while the `insertBlockedRow` and `insertRegistration` helpers above have the order `t, dbMap, fc`. I don't have a strong preference one way or the other, but it should be consistent.